napari/_tests/test_viewer.py

```diff
@@ -137,3 +137,54 @@ def test_changing_theme(viewer_factory):
 
     with pytest.raises(ValueError):
         viewer.theme = 'nonexistent_theme'
+
+
+@pytest.mark.parametrize('layer_class, data, ndim', layer_test_data)
+def test_roll_traspose_update(viewer_factory, layer_class, data, ndim):
+    """ Controlling that translation and rolling preserve correct sequence
+    of translate and scaling values.
+    """
+
+    def check_view_consistency(layer, viewer, transf_dict):
+        """ Utility function for doing the checks.
+        """
+        # Get an handle on visual layer:
+        vis_lyr = viewer.window.qt_viewer.layer_to_visual[layer]
+
+        # Visual layer attributes should match expected from viewer dims:
+        for transf_name, transf in transf_dict.items():
+            disp_dims = viewer.dims.displayed  # dimensions displayed in 2D
+            vis_vals = getattr(vis_lyr, transf_name)[
+                1::-1
+            ]  # values of visual layer
+
+            # If layer's a pyramid scale is multiplued by a factor 2:
```

Comment:
Interesting, I think it makes sense to let this go in now, as I am working on a fix for some pyramid stuff in #1053 that can address this in more detail

Reply:
Though maybe you could fix `multiplued ` to `multiplied`

Reply:
ok!